test/models/project_test.rb

```diff
@@ -238,7 +238,27 @@
       output.string.include?('using the error callback').must_equal(true)
       output.string.include?("Can't get here").must_equal(false)
     end
-
   end
 
+  describe 'deploy group releases' do
```

Comment:
Aye, tough battle with keeping it consistent with existing code, but yea, since there is no prevailing pattern, I'll do this.